### ISSUE TITLE
DNS A record triggers warning when deploying

### ISSUE THREAD

**Issue by geigerzaehler** [radicle-dev/radicle-run#40]
When deploying this site I receive an email with the following warning

> Your site's DNS settings are using a custom subdomain, registry.radicle.xyz, that's set up as an A record. We recommend you change this to a CNAME record pointing at radicle-dev.github.io. For more information, see https://help.github.com/en/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site.

Should we change the DNS A record for `registry.radicle.xyz` to `CNAME radicle-dev.github.io`?

**Comment by NunoAlexandre** [comment]
We have both the A records and the CNAME in place.
![dns](https://user-images.githubusercontent.com/7075260/81063683-9b6e5200-8ed8-11ea-980b-c4ce011a699c.png)

It seems like the warning things that we have the A records, therefore not the CNAME.

When the domain for this githug-pages site was `radicle.run`, a proper domain, we needed both. Now that is `registry.radicle.xyz`, a sub-domain, the A records don't seem necessary anymore.

So the question really is: Should we drop the `A` records? I'm actually surprised the warning suggests them to not be necessary for sub-domains. Now, we can delete the A records, but the only disadvantage I see in keeping them are these warnings. I didn't delete those records initially because I thought they would still be necessary and then because having them in place makes migration from sub-domain to domain effortless, whereas if we delete them we would need to lookup those IPs again and add the A records back.

**Comment by geigerzaehler** [comment]
It looks like using an `A` record with a `CNAME` record removes the `CNAME` record
~~~
$ drill CNAME registry.radicle.xyz
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 55000
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0 
;; QUESTION SECTION:
;; registry.radicle.xyz.        IN      CNAME

;; ANSWER SECTION:

;; AUTHORITY SECTION:
radicle.xyz.    85      IN      SOA     ns-cloud-b1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300

;; ADDITIONAL SECTION
~~~
The answer section is empty.

I’d suggest dropping the `A` records so that we don’t always receive that warning.

**Comment by NunoAlexandre** [comment]
I just removed the A records and the site went right down:

![Screenshot from 2020-05-05 15-22-17](https://user-images.githubusercontent.com/7075260/81070984-56501d00-8ee4-11ea-9887-9ae75eb5bc15.png)

Either the CNAME suggestion they give doesn't work since it could be already taken for `radicle.xyz`, or I don't know. I don't think it was a matter of dns propagation. For now, I set the A records back.
